paf-mvp-operator-express/src/operator-api.ts

```diff
@@ -70,12 +70,14 @@ export const addOperatorApi = (
   name: string,
   keys: KeyInfo[],
   allowedDomains: AllowedDomains,
+  dpoEmailAddress: string,
+  privacyPolicyUrl: URL,
   s2sOptions?: AxiosRequestConfig
```

Comment:
FAR: Too many arguments. Need to be refactored

Reply:
Totally agree.